### PR TITLE
Update plugin.json

### PR DESCRIPTION
**Pull request by trib0r3**
Regarding #7

### CODE REVIEW

**Comment by trib0r3** [comment]
@thebabush you still need to create a release based on tag and file an issue on binja-community repo https://binary.ninja/2019/07/04/plugin-manager-2.0.html#6-submitting-your-plugin

**Comment by thebabush** [comment]
I don't have an active binja license atm, so I don't feel comfortable submitting my untested plugin. If you tell me it still works I can submit it.

**Comment by thebabush** [comment]
Ah I'm dumb (sorry, just finished 2 days of CTF). In the upcoming days I'll take a look :) thanks!

**Comment by trib0r3** [comment]
> I don't have an active binja license atm, so I don't feel comfortable submitting my untested plugin. If you tell me it still works I can submit it.

Hello, it seems to work fine, but if you don't feel comfortable about publishing plugin without binja license I can do it from mine fork ;)

**Comment by thebabush** [comment]
I mean, I can submit it, I just wanted to have confirmation.

Do they really require the README to be in a specific format? That I don't like, especially because this repo can be used in a normal python interpreter independently from binja.

**Comment by trib0r3** [comment]
No, I just recreated it from auto-readme script, I can revert the README to the previous version if you want

File: plugin.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-    }
-  }
-}
\ No newline at end of file
+  "pluginmetadataversion": 2,
+  "name": "Nampa",
+  "type": ["core", "binaryview"],
+  "api": ["python3"],
+  "description": "FLIRT for (binary) ninjas",
+  "longdescription": "Plugin for using FLIRT signatures inside of Binary Ninja. Most of the code is a port from radare's implementation.",
```
Comment (thebabush) [inline plugin.json]: maybe you can remove the "most of the code ..." part.

File: plugin.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+  "license": {
+          "name": "LGPL v3",
+          "text": "This library is free software; you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation; either version 3.0 of the License, or (at your option) any later version.\n\nThis library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.\n\nYou should have received a copy of the GNU Lesser General Public License along with this library; if not, write to the Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA"
+  },
+  "platforms": ["Windows", "Linux", "Darwin"],
+  "installinstructions" : {
+          "Windows":"```\npip install -r requirements.txt\n```",
+          "Linux":"Check [Windows](#windows) installation",
```
Comment (thebabush) [inline plugin.json]: I guess it's nicer to copy paste the command instead of using a markdown link.

File: plugin.json [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+          "Darwin":"Check [Windows](#windows) installation"
+  },
+  "dependencies": {
+      "pip": ["future"]
```
Comment (thebabush) [inline plugin.json:19]: if binja is now python 3, future shouldn't be needed anymore maybe?

File: plugin.json [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+  },
+  "version": "1.0",
+  "author": "Paolo Montesel (github.com/kenoph)",
```
Comment (thebabush) [inline plugin.json]: a while ago I changed my old github account name to `thebabush`

**Review by thebabush — COMMENTED** [review]
So, if you can revert the readme (I'll update it later) and do the small changes I annotated below, I'll happily merge it and add you to a brand new "thanks" section in the readme.
If you don't have time to test it, don't worry about the `future` requirement... We can simply leave it there.

**Comment by trib0r3** [comment]
Generally plugin seems to work, I will apply mentioned changes later today :)

**Comment by thebabush** [comment]
If you are busy, just revert the readme and then I'll fix the json myself ;)

EDIT: btw, please squash the revert commit.

**Comment by trib0r3** [comment]
@thebabush ok, it's done

**Comment by thebabush** [comment]
I'll submit the plugin later. Thank you very much for your effort!

**Comment by trib0r3** [comment]
Meh, I just checked more features of this plugin and unfortunately some things are not working as they should. I will make more PR with fixes :/

**Comment by thebabush** [comment]
Some links to the sig files are broken for sure, but yeah I guess they changed the API quite a bit since last time I used binja.

**Comment by trib0r3** [comment]
Actually binja API seems to work correctly, I fixed all issues already, they were related to small py2/py3 changes, check the newest PR ;)

**Pull request merged**